### PR TITLE
[Customer] Added phone number to customer

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT
| Doc PR          |

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
Migration file is missing.

**Comment by aramalipoor** [comment]
:+1:

File: src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/Customer.orm.xml [hunk 1]
```diff
@@ -61,6 +61,8 @@
             </join-table>
         </many-to-many>
 
+        <field name="phoneNumber" column="phone_number" type="string" nullable="true" />
```
Comment (Zales0123) [inline src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/Customer.orm.xml]: Don't we need any validation configured for this field?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/Customer.orm.xml]: We will add it later when we cover this with scenarios in frontend.

**Comment by pjedrzejewski** [comment]
Thanks Grzesiu!

**Comment by gperdomor** [comment]
Sorry, I'm late but this is my opinion:

What happen if we need handle multiple phone numbers?, mobile and home number for example. In that case, i don't like the idea of creation of a new entity <code>phoneNumber</code>, at least not in <code>UserBundle</code>, i think this should be managed this in the <code>CoreBundle</code> and leave the <code>UserBundle</code> more generic possible.

**Comment by peteward** [comment]
I think this is a good change.

Customers have Phone numbers in the same way that they have email addresses, names and addresses. It's definitely not a Core attribute to say that a customer has a phone number.

If you want multiple phone numbers then perhaps this is an extension of your own on top of the existing phoneNumber?

**Pull request merged**